docs-markdown/snippets/snippets.json

```diff
@@ -23,7 +23,7 @@
   "icon": {
     "prefix": "image",
     "body": [
-      ":::image type=\"icon\" source=\"\":::"
+      ":::image type=\"icon\" source=\"\" border=\"false\" :::"
```

Comment:
Hey @adunndevster , there's a space before the closing triple colon.  Should that be removed?

Reply:
I'll remove it, but it does not cause any harm :)